cfme/containers/provider/openshift.py

```diff
@@ -412,3 +441,31 @@ def is_refreshed(self, refresh_timer=None, refresh_delta=600):
             )
         finally:
             return valid
+
+    def enable_provider_alerts_collection(self):
+        provider_profile = \
+            self.appliance.collections.alert_profiles.instantiate(ProviderAlertProfile,
+                                                                  "Prometheus Provider Profile")
+
+        provider_profile_view = navigate_to(provider_profile, 'Edit assignments')
+        if provider_profile_view.assign_to.selected_option != "The Enterprise":
+            provider_profile_view.assign_to.select_by_visible_text("The Enterprise")
+            provider_profile_view.save_button.click()
+        else:
+            provider_profile_view.cancel_button.click()
+
+    def enable_node_alerts_collection(self):
+        node_profile = \
```

Comment:
Please don't use `\` line continuation when it can be avoided.  Consider shortening `node_profile` to just `profile` since its the only one in this methods scope.

```python
        node_profile = self.appliance.collections.alert_profiles.instantiate(
            NodeAlertProfile, "Prometheus node Profile")
        node_profile = self.appliance.collections.alert_profiles.instantiate(NodeAlertProfile,
                                                                                                                "Prometheus node Profile")